src/views/Maintenance.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div>
+    <div class="maintenance container wrapper mt-2">
+      <div class="p-5 content text-center">
+        <div v-if="view === 'recharge'">
+          <div class="recharge">
+            <h1>We're recharging our powers.</h1>
+            <div class="gif" />
```

Comment:
why not simply use img? using background images instead of image comes with a lot of trade-offs, best practice is to avoid it. To center the image inside the constraint/fixed size, add a class to it and apply a property i.e. `object-fit: cover`. Also please use more relevant naming - i.e. maintenance--image (though naming convention is not yet agreed). Anyway .gif, .gif2 is quite inappropriate, image format might change, and even now .gif2 contains a svg.